### ISSUE TITLE
compiling recode bleeding edge: autoconf tools seem to have blind spots

### ISSUE THREAD

**Issue by sblondeel** [rrthomas/recode#32]
While reporting #31 a few hours ago I noticed I could not compile recode on my Debian10 (stable) with very few development tools installed.

The bootstrap script detected I missed some tools and I had to install the following packages to please it:

```
m4 libtool help2man texinfo autopoint
```
However the build step still failed after that (error message is quoted in #31).

@rrthomas kindly explained me the empty command name should have been "msgfmt" which gave me the clue to install yet another Debian package:

```
 gettext
```

Now I get another error message, probably later in the build process:

```
Completing strip-data.c
Writing strip-pool.c
echo '#include "config.h"' > merged.c
(cd . && cat ascilat1.l iso5426lat1.l ansellat1.l ltexlat1.l btexlat1.l txtelat1.l ) \
| /usr/bin/python3 ./mergelex.py > merged.tm1
: -t -8 -Plibrecode_yy merged.tm1 > merged.tm2
grep -av '^# *line [0-9]' merged.tm2 >> merged.c
make[2]: *** [Makefile:2159: merged.c] Error 1
make[2]: Leaving directory '/tmp/recode/src'
make[1]: *** [Makefile:1509: all-recursive] Error 1
make[1]: Leaving directory '/tmp/recode'
make: *** [Makefile:1441: all] Error 2
```

so I believe I miss yet another command / package.

May I suggest the bootstrap script to test for gettext and whatever I am still missing at this point?

**Comment by rrthomas** [comment]
That next missing command is `flex`. I'll add it to the list in `bootstrap.conf`, along with gettext.

**Comment by sblondeel** [comment]
I just tested again and it is now working, so I could check myself the fix to #31. Thanks!
